pyproject.toml

```diff
@@ -89,19 +89,16 @@ target-version = "py310"
 
 [tool.ruff.lint]
 # TODO Comment in more of the below as we clean up the code.
-ignore = [
```

Comment:
This seems like its reverting a change from an earlier PR, which I assume is not intended.

Reply:
Ahh damn, good spot.

Reply:
Out of interest, what does this ignore do?